backend/src/federation/client/1_0/logging/PublicCommunityInfoLogging.view.ts

```diff
@@ -11,7 +11,7 @@ export class PublicCommunityInfoLoggingView extends AbstractLoggingView {
     return {
       name: this.self.name,
       description: this.self.description,
-      creationDate: this.dateToString(this.self.creationDate),
+      creationDate: this.self.creationDate,
```

Comment:
The creationDate Attribute from PublicCommunityInfo is declared as Date so this function dateToString from AbstractLoggingView 
```ts
public dateToString(date: Date | undefined | null): string | undefined {
    if (date) {
      return date.toISOString()
    }
    return undefined
  }
  ```
  is used to make it pretty for logging.
  Or is it a bug in PublicCommunityInfo that it contains creationDate sometimes as string? That it is maybe better, to update the interface, ins't it?